specification/network/resource-manager/Microsoft.Network/stable/2020-04-01/virtualWan.json

```diff
@@ -3648,6 +3648,69 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{groupName}/providers/Microsoft.Network/virtualHubs/{virtualHubName}/effectiveRoutes": {
+      "post": {
+        "operationId": "EffectiveVirtualHubRoutes",
+        "x-ms-examples": {
+          "EffectiveVirtualHubRouteList": {
+            "$ref": "./examples/EffectiveVirtualHubRouteList.json"
+          }
+        },
+        "description": "Gets the effective routes for the Virtual Hub resource or the specified resource .",
+        "parameters": [
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The resource group name of the VirtualHub."
+          },
+          {
+            "name": "virtualHubName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the VirtualHub."
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "effectiveRoutesParameters",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "$ref": "#/definitions/EffectiveRoutesParameters"
+            },
+            "description": "Parameters supplied to get the effective routes."
+          }
+        ],
+        "responses": {
+          "200": {
```

Comment:
i dont know what we return for post. Need to still try it out in Slice. Can you confirm with Shivam/Farhan